__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -364,7 +364,7 @@ describe('TESTING ChatSchema', () => {
         message: 'message',
         rawJson: expect.any(String),
         status: 'sent',
-        type: 'regular',
+        type: expect.any(String),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded 'regular' to `expect.any(String)` in the test might make the test less strict. If the type of message is important and should be specifically 'regular', this change could allow incorrect types to pass the test.
```diff
-        type: 'regular',
+        type: expect.any(String),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->